how-to/register-with-home/client/src/launch.ts

```diff
@@ -1,19 +1,21 @@
 import { getCurrentSync } from '@openfin/workspace-platform';
 import { App } from "@openfin/workspace";
 import { fin } from 'openfin-adapter/src/mock';
+import { launchView } from './browser';
 
 export async function launch(appEntry: App) {
     console.log("Application launch requested: ", appEntry);
-    if(appEntry.manifestType === "external") {
+    if (appEntry.manifestType === "external") {
         console.log("Application requested is a native app. Managing request via platform and not Workspace.");
         let options: OpenFin.ExternalProcessRequestType = {};
         options.path = appEntry.manifest;
         options.uuid = appEntry.appId
-
         await fin.System.launchExternalProcess(options);
+    } else if (appEntry.manifestType === "direct") {
+        launchView(appEntry.manifest)
```

Comment:
if we provide the details directly in the apps list, we can launch the view instead of launching the app (launching app use the manifest url to go to the inner details). This way we bypass the need to go to the url and see for the application data.